courses/views.py

```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Count
 
-# Create your views here.
+from courses.models import Category, SubCategory, Course
+
+class ListView(View):
+    def get(self, request):
+        category_id     = request.GET.get('category', None)
+        sub_category_id = request.GET.get('sub_category', None)
+        course_list     = []
+        # 카테고리 분류
+        if category_id:
+            course_list = Course.objects.filter(
+                sub_category__in = SubCategory.objects.filter(
+                    category = Category.objects.get(id=category_id)
```

Comment:
```suggestion
            course_list = Course.objects.filter(sub_category__category_id = category_id
```

이런식으로 줄여서 가독성을 높일 수 있겠네요!